.github/workflows/continuous_integration.yml

```diff
@@ -32,10 +32,8 @@ jobs:
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Lint with flake8
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # The GitHub editor is 127 chars wide
+          flake8 . --count --max-line-length=127 --statistics
```

Comment:
127 seems pretty long, where did you get this number?

Reply:
127 is the GitHub editor length. I changed the length to now match the recommended length in Pycharm, 120 chars.